interfaces/console/src/app/configure/page.tsx

```diff
@@ -38,10 +38,12 @@ const Page = () => {
     setIsChecked(isChecked);
   };
   const onInstallProgressComplete = () => {
-    const id = 'uk-sa9001-tnode-a1-1234';
-    router.push(`/configure/node/${id}?lat=-4.322447&lng=15.307045`);
-  };
+    const id = searchParams.get('nodeId') ?? 'uk-sa9001-tnode-a1-1234';
+    const lat = searchParams.get('lat') ?? '-4.322447';
+    const lng = searchParams.get('lng') ?? '15.307045';
```

Comment:
We need to remove dummy nodeId and lat, lng.  If any of these param missing we can't continue with the configure flow. We can just show alert snackbar.